.gitignore

```diff
@@ -1,5 +1,20 @@
-node_modules
-.idea
-package-lock.json
```

Comment:
Just calling this out for other reviewers: we want reproducible builds, so we're using `package-lock.json` for our _build tooling_. By default, Lerna doesn't create `package-lock.json` files for the different packages, since those aren't typically published to npm anyways.

Reply:
Exactly, if you don't have lerna as a global dependency then you should run `npm install && lerna bootstrap` in the root of the repo to install all the dependencies.